proxy/common/_compat.py

```diff
@@ -1,9 +1,19 @@
-"""Compatibility code for using Proxy.py across various versions of Python.
+# -*- coding: utf-8 -*-
+"""
+    proxy.py
+    ~~~~~~~~
+    ⚡⚡⚡ Fast, Lightweight, Pluggable, TLS interception capable proxy server focused on
+    Network monitoring, controls & Application development, testing, debugging.
+
+    :copyright: (c) 2013-present by Abhinav Singh and contributors.
+    :license: BSD, see LICENSE for more details.
+
+    Compatibility code for using Proxy.py across various versions of Python.
```

Comment:
With this change, you're breaking the PEP 257 style compatibility.

Reply:
Yep :(. I think will be addressed with doc-string fix.  `Compatibility code for using Proxy.py across various versions of Python.` will then be the only content of doc-string

Reply:
Tracking here https://github.com/abhinavsingh/proxy.py/issues/776